scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -2732,10 +2730,6 @@ def check_containers(self, run_state):
                 for cid, info in six.iteritems(running_containers):
                     pod = None
                     if "k8s_info" in info:
-                        pod_name = info["k8s_info"].get("pod_name", "invalid_pod")
-                        pod_namespace = info["k8s_info"].get(
-                            "pod_namespace", "invalid_namespace"
-                        )
```

Comment:
These values are used below in the warning that is now commented out (line 2738).  It's not clear to me if we should just remove that line or not, but just in the interest of kicking the can down the road, how about you inline the `info["k8s_info"].get("pod_name", "invalid_pod")` and `info["k8s_info"].get("pod_namespace", "invalid_namespace")` into the log line (i.e., don't use the variable names).

Reply:
Yeah, agreed, will make that change.

Reply:
Agreed, added it back in f6e2f6d and commented it out.